PingProtect/PingProtect/Sources/PIProtect.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  PIProtect.swift
+//  PingProtect
+//
+//  Copyright (c) 2024 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import Foundation
+import FRAuth
+import PingOneSignals
+
+/*
+ PIProtect is for initializing and interacting with Ping Protect SDK
+ */
+public final class PIProtect: NSObject {
+    
+    
+    /// Inititialize Ping Protect SDK
+    /// - Parameters:
+    ///   - initParams: `PIInitParams` containing parameters for the init
+    ///   - callback: Result callback of the init
+    public static func initSDK(initParams: PIInitParams? = nil, callback: @escaping (Error?) -> Void) {
```

Comment:
This is totally non-blocking, and I'm going to approve this PR regardless, but one thing we could do with `init` used on the other platforms and `initSDK` on this one is we could all just use `initialize` 🤷

Reply:
I agree @cerebrl!